viewer/packages/pointclouds/wasm/src/test_utils/mod.rs

```diff
@@ -0,0 +1,7 @@
+/// Takes a u32, perturbs it, and scales it down to a
+/// number between -1.0
+pub fn normalize_coordinate(a: u32) -> f64 {
+    const COORD_GRANULARITY: i32 = 100_000;
+
+    ((a % (COORD_GRANULARITY as u32)) as f64 / (COORD_GRANULARITY as f64)) * 2.0 - 1.0
+}
```

Comment:
What is this?

Reply:
It's a method for normalizing a `u32` to a number between -1 and 1.

Reply:
Update: Removed all of this.. Turns out there's a `rng.gen_range` method that's much more handy for these things.